base/silent/src/test/java/org/openscience/cdk/silent/AtomTest.java

```diff
@@ -72,6 +72,91 @@ public void testAtom_String() {
     }
 
     @Test
+    public void testAtom_NH4plus_direct() {
```

Comment:
I guess this method is now in two FooAtomTest classes? Have you tried placing it in AbstractAtomTest? That was created to have tests that should be run for all IAtom implementations...